CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Added
 - Added summary endpoint for annotation groups to list the number of labels with different qualities (YES, NO, MISS, UNSURE) to support annotation applications [\#4221](https://github.com/raster-foundry/raster-foundry/pull/4221)
 - Added project histogram support for COG and Avro scenes [\#4190](https://github.com/raster-foundry/raster-foundry/pull/4190)
+- Added semi-fancy error-handling to backsplash [\#4258](https://github.com/raster-foundry/raster-foundry/pull/4258)
```

Comment:
I think this can be improved a little bit, ideally this is something that can be copy/pasted to users with minimal editing if we needed/wanted to